model_compression_toolkit/core/pytorch/back2framework/instance_builder.py

```diff
@@ -38,12 +38,16 @@ def node_builder(n: BaseNode) -> Module:
     return node_instance
 
 
-def identity_wrapper(node: BaseNode, module: Module):
+# todo: remove. It is not used anymore
```

Comment:
Is this todo referring to this current PR?

Reply:
No, in a following PR